packages/flame/test/effects/opacity_effect_test.dart

```diff
@@ -242,11 +242,7 @@ void main() {
         game.update(dt);
       }
       game.update(1000 - totalTime);
-      // TODO(spydon): The loop above has an average of 100fps.
-      // It should change from 0-255 in 1s so it will change alpha with an
-      // average of 255/100=2.5 per tick, which should not result in a need of
-      // an epsilon value this high.
-      expectDouble(component.getOpacity(), 1.0, epsilon: 100 * _epsilon);
+      expectDouble(component.getOpacity(), 1.0, epsilon: _epsilon);
```

Comment:
Ah exactly, this was it.

Reply:
It appears the test now works much more reliably after switching to opacities. At least the epsilon is much smaller.